.travis/custom_release.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Do we need this script file anymore after containerization?

Reply:
this is an accident. It seems there was a mistake i missed when rebasing against master